database/youtube/web-development.json

```
@@ -334,5 +334,13 @@
     "category": "youtube",
     "subcategory": "web-development",
     "language": "english"
+  },
+  {
+    "name": "developedbyed",
+    "description": "This channel has a wide variety of educational videos on technologies like HTML,CSS,Javascript,Tailwind CSS and React",
```

[proofreading/suggestion] Since you mentioned "Tailwind", I highly recommend describing regular CSS to avoid confusion. Also, always use a comma when describing more than one thing.
```suggestion
    "description": "This channel has a wide variety of educational videos on technologies like HTML,vanilla CSS,Javascript,Tailwind CSS,and React",
```